tests/test_reporter.py

```diff
@@ -34,6 +35,7 @@ def request_get_mocked(filename):
             filename: Asset's filename.
         """
         get_mocked = mock.Mock()
+
```

Comment:
This is a needed change, but it's not about complying with the line length requirements as commit subject says.
Please either make it a separate commit, or change the commit subject to something like "Make tests comply with pep8 requirements". While this is a small matter, it's easier to simply always keeping commit descriptions true, than deciding every time if it's important or not.

Reply:
You are so right. I have completely forgot that I have been running pep8 locally... Please take a look to the last push, I have added pep8 and remove the .pylintrc file to keep it simple as possible.

Reply:
Well I've changed it by pycodestyle because pep8 is deprecated as you can see [here](https://travis-ci.org/RH-FMK/skt/builds/353475444?utm_source=github_status&utm_medium=notification)